client/src/Component/MeetingComponent/MeetingComponent.css

```diff
@@ -0,0 +1,82 @@
+.meeting_box{
+  border-radius: .4em;
+  margin: 5px 20px 5px;
+}
+
+.meeting_orange{
+  background: orange;
+  width: 100%;
+  height: 5px;
+  margin: 0px;
+  border-top-left-radius: .4em;
+  border-top-right-radius: .4em;
+}
+
+.meeting_blue{
+  background: blue;
+  width: 100%;
+  height: 5px;
+  margin: 0px;
+  border-top-left-radius: .4em;
+  border-top-right-radius: .4em;
+}
```

Comment:
There is a way to combine these. You can use something like `https://www.npmjs.com/package/classnames` to pass multiple styles to the same component. Now what you can do is break these into something like:
```
const useStyles = makeStyles({
   tagBar: {
        width: "100%";
        height: 5;
        margin: 0;
        border-top-left-radius: ".4em";
        border-top-right-radius: ".4em";
    },
    orangeBackground: {
        background: "orange";
    },
    blueBackground: {
        background: "blue";
    }
});
```
etc.
and then do <Component className={classNames(classes.tagBar, classes.backgroundOrange)} />

The goal is to get rid of this css file, and have all of your styling in makeStyles.